src/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const swaggerUi = require('swagger-ui-express');
+const swaggerDocument = require('./swagger.json');
+
+const app = express();
+const PORT = 3000;
+
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+
+app.get('/health-check', (req, res) => {
+	res.json({ status: 'Server is running' });
+});
+
+app.use('/not-found', (req, res) => {
```

Comment:
For a more general and standardized approach, it's preferable to use this option:
app.use((req, res, next) => {
  res.status(404).json({ error: 'Not Found' });
});
It's a cleaner and more maintainable solution when error handling is desired for undefined routes.